test/actions.test.ts

```diff
@@ -67,185 +58,129 @@ describe("Test Custom SDK Logic", () => {
-      const domains: Domain[] = await dataStoreApiClient.getSubdomainsById(
-        domainThatHasSubdomains
-      );
-      expect(domains.length).to.not.eq(0);
-    });
-    it("Returns empty array for domains that have no subdomains", async () => {
-      const domains: Domain[] = await dataStoreApiClient.getSubdomainsById(
-        domainThatHasNoSubdomains
-      );
-      expect(domains.length).to.eq(0);
-    });
-  });
-
-  describe("get domains", () => {
-    it("gets most recent domains", async () => {
-      const sdkInstance = zNSSDK.createInstance(config);
-      const domains = await sdkInstance.getMostRecentDomains(10, 0);
-      expect(domains.length).to.equal(10);
-    });
-    it("cannot get over 5000 most recent domains", async () => {
-      const sdkInstance = zNSSDK.createInstance(config);
-      expect(sdkInstance.getMostRecentDomains(5000, 0)).to.eventually.throw(
-        Error
-      );
-    });
-    it("gets most recent subdomains", async () => {
-      const sdkInstance = zNSSDK.createInstance(config);
-      const domains = await sdkInstance.getMostRecentSubdomainsById(
-        wilderDogsDomainId,
-        2,
-        0,
-        false
-      );
-      expect(domains.length).to.equal(2);
-    });
-    it("gets most recent subdomains via the data store", async () => {
-      const sdkInstance = zNSSDK.createInstance(config);
-      const domains = await sdkInstance.getMostRecentSubdomainsById(
-        wilderDogsDomainId,
-        2,
-        0,
-        true
-      );
-      expect(domains.length).to.equal(2);
-    });
-    it("cannot get over 5000 most recent subdomains", async () => {
-      const sdkInstance = zNSSDK.createInstance(config);
-      expect(
-        sdkInstance.getMostRecentSubdomainsById(wilderDogsDomainId, 5000, 0, false)
-      ).to.eventually.throw(Error);
-    });
-  });
-
-  describe("content moderator", () => {
-    it("flags inappropriate content", async () => {
-      var sample = `booty`;
-      const sdkInstance = zNSSDK.createInstance(config);
-      const moderation = await sdkInstance.utility.checkContentModeration(
-        sample
-      );
-      expect(moderation.flagged).to.be.true;
-      expect(moderation.reason).to.equal("Contains explicit content.");
-    });
-
-    it("flags special characters", async () => {
-      var sample = `2 Chainz!`;
-      const sdkInstance = zNSSDK.createInstance(config);
-      const moderation = await sdkInstance.utility.checkContentModeration(
-        sample
-      );
-      expect(moderation.flagged).to.be.true;
-      expect(moderation.reason).to.equal("Contains special characters.");
-    });
-
-    it("does not flag acceptable content", async () => {
-      var sample = `2Chains`;
-      const sdkInstance = zNSSDK.createInstance(config);
-      const moderation = await sdkInstance.utility.checkContentModeration(
-        sample
-      );
-      expect(moderation.flagged).to.be.false;
+  describe("E2E with Goerli", () => {
```

Comment:
I assume I should have more test cases to test for `getDomainById`, `getDomainsByName`, `getAllDomains`, etc ... @JamesEarle